### PR TITLE
Changes for add-ons

### PR DESCRIPTION
**Pull request by beckykd**
The previous PR got too messy with conflicts, so reopened it.

### CODE REVIEW

**Comment by jenglick** [comment]
@beckykd quick note that we are using the nonhyphenated version, "Qiskit addons", could you update that throughout? thank you!

File: docs/guides/qiskit-add-ons.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: Qiskit add-ons
+description: Understand the Qiskit add-on tools , which help you build utility-grade quantum workflows.
```
Comment (garrison) [inline docs/guides/qiskit-add-ons.mdx:3]:
```suggestion
description: Understand the Qiskit add-on tools, which help you build utility-grade quantum workflows.
```

File: docs/guides/qiskit-add-ons.mdx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- Read the [documentation.](https://qiskit.github.io/qiskit-addon-obp)
+
+### Circuit cutting
+
+Circuit cutting reduces the depth of transpiled circuits by decomposing long range gates.
+
+- Visit the [GitHub repository.](https://github.com/Qiskit/qiskit-addon-cutting)
+- Read the [documentation.](https://qiskit.github.io/qiskit-addon-cutting)
+
+## Postprocessing tools
```
Comment (garrison) [inline docs/guides/qiskit-add-ons.mdx:35]: This is hyphenated at https://docs.quantum.ibm.com/guides/intro-to-patterns.

```suggestion
## Post-processing tools
```

File: docs/guides/qiskit-add-ons.mdx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+---
+title: Qiskit add-ons
+description: Understand the Qiskit add-on tools , which help you build utility-grade quantum workflows.
+---
+
+# Qiskit add-ons
+
+Qiskit add-ons are a collection of modular tools for building utility-scale workloads powered by Qiskit. These tools are derived from advanced research capabilities and are designed to be stackable with other tools as part of a [Qiskit pattern.](./intro-to-patterns)
```
Comment (jenglick) [inline docs/guides/qiskit-add-ons.mdx:8]: let's replace these two intro sentences with:

> Qiskit addons are a collection of research capabilities for enabling algorithm discovery at the utility scale. These capabilities build upon Qiskit’s performant foundation of tools for creating and running quantum algorithms. They are provided as modular software components that can [plug into a workflow](./intro-to-patterns) to scale or design new quantum algorithms.

File: docs/guides/qiskit-add-ons.mdx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+### Operator backpropagation
+
+Operator backpropagation (OBP) reduces circuit depth by trimming operations from the end at the cost of more operator measurements.
+
+- Visit the [GitHub repository.](https://github.com/Qiskit/qiskit-addon-obp)
+- Read the [documentation.](https://qiskit.github.io/qiskit-addon-obp)
+
+### Circuit cutting
+
+Circuit cutting reduces the depth of transpiled circuits by decomposing long range gates.
```
Comment (jenglick) [inline docs/guides/qiskit-add-ons.mdx:30]: replace "long rate gates" with:

"... entangling gates between non-adjacent qubits."

File: docs/guides/qiskit-add-ons.mdx [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+- Visit the [GitHub repository.](https://github.com/Qiskit/qiskit-addon-cutting)
+- Read the [documentation.](https://qiskit.github.io/qiskit-addon-cutting)
+
+## Postprocessing tools
+
+### Sample-based quantum diagonalization
+
+Sample-based quantum diagonalization (SQD) classically postprocesses noisy quantum samples to yield more accurate energy estimations.
```
Comment (jenglick) [inline docs/guides/qiskit-add-ons.mdx:39]: "... to yield more accurate energy estimations of quantum system Hamiltonians, for example in chemistry applications."

File: docs/guides/qiskit-add-ons.mdx [hunk 6]
```diff
@@ -0,0 +1,42 @@
+---
+title: Qiskit add-ons
+description: Understand the Qiskit add-on tools , which help you build utility-grade quantum workflows.
+---
+
+# Qiskit add-ons
+
+Qiskit add-ons are a collection of modular tools for building utility-scale workloads powered by Qiskit. These tools are derived from advanced research capabilities and are designed to be stackable with other tools as part of a [Qiskit pattern.](./intro-to-patterns)
+
+## Mapping tools
```
Comment (jenglick) [inline docs/guides/qiskit-add-ons.mdx:10]: replace "tools" with "addons"

File: docs/guides/qiskit-add-ons.mdx [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+### Multi-product formulas
+
+Multi-product formulas (MPF) reduce the Trotter error of Hamiltonian dynamics through a weighted combination of several circuit executions.
+
+- Visit the [GitHub repository.](https://github.com/Qiskit/qiskit-addon-mpf)
+- Read the [documentation.](https://qiskit.github.io/qiskit-addon-mpf)
+
+## Optimizing tools
```
Comment (jenglick) [inline docs/guides/qiskit-add-ons.mdx:19]: replace "tools" with "addons"

File: docs/guides/qiskit-add-ons.mdx (hunk 2 above)
Comment (jenglick) [inline docs/guides/qiskit-add-ons.mdx:35]: replace "tools" with "addons"

**Review by jenglick — COMMENTED** [review]
@beckykd some more minor changes. thank you!

File: docs/guides/qiskit-addons.mdx [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+## Mapping addons
+
+### Multi-product formulas
+
+Multi-product formulas (MPF) reduce the Trotter error of Hamiltonian dynamics through a weighted combination of several circuit executions.
+
+- Visit the [GitHub repository.](https://github.com/Qiskit/qiskit-addon-mpf)
+- Read the [documentation.](https://qiskit.github.io/qiskit-addon-mpf)
+
+## Optimizing addons
```
Comment (jenglick) [inline docs/guides/qiskit-addons.mdx:19]:
```suggestion
## Addons for optimizing
```

File: docs/guides/qiskit-addons.mdx [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+- Read the [documentation.](https://qiskit.github.io/qiskit-addon-obp)
+
+### Circuit cutting
+
+Circuit cutting reduces the depth of transpiled circuits by decomposing entangling gates between non-adjacent qubits.
+
+- Visit the [GitHub repository.](https://github.com/Qiskit/qiskit-addon-cutting)
+- Read the [documentation.](https://qiskit.github.io/qiskit-addon-cutting)
+
+## Post-processing addons
```
Comment (jenglick) [inline docs/guides/qiskit-addons.mdx:35]:
```suggestion
##  Addons for post-processing
```

**Review by jenglick — COMMENTED** [review]
Becky, switching up the subsection headers on you again ; )

**Pull request closed without merging**